src/main/java/org/gridsuite/shortcircuit/server/ShortCircuitController.java

```diff
@@ -97,15 +93,16 @@ public ResponseEntity<ShortCircuitAnalysisResult> getResult(@Parameter(descripti
                 : ResponseEntity.notFound().build();
     }
 
-    @GetMapping(value = "/results/{resultUuid}/fault_results/paged", produces = APPLICATION_JSON_VALUE)
-    @Operation(summary = "Get a fault results page for a given short circuit analysis result")
-    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The page of fault results"),
+    @GetMapping(value = "/results/{resultUuid}/paged", produces = APPLICATION_JSON_VALUE)
+    @Operation(summary = "Get a results page for a given short circuit analysis result")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The page of results"),
         @ApiResponse(responseCode = "404", description = "Short circuit analysis result has not been found")})
-    public ResponseEntity<Page<FaultResult>> getPagedFaultResults(@Parameter(description = "Result UUID") @PathVariable("resultUuid") UUID resultUuid,
-                                                                @Parameter(description = "Full or only those with limit violations or none fault results") @RequestParam(name = "mode", required = false, defaultValue = "WITH_LIMIT_VIOLATIONS") FaultResultsMode mode,
-                                                                Pageable pageable) {
-        Page<FaultResult> faultResultsPage = shortCircuitService.getFaultResultsPage(resultUuid, mode, pageable);
-        return faultResultsPage != null ? ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(faultResultsPage)
+    public ResponseEntity<ShortCircuitAnalysisPagedResults> getPagedResults(@Parameter(description = "Result UUID") @PathVariable("resultUuid") UUID resultUuid,
+                                                                            @Parameter(description = "Full or only those with limit violations or none fault results") @RequestParam(name = "mode", required = false, defaultValue = "WITH_LIMIT_VIOLATIONS") FaultResultsMode mode,
+                                                                            @Parameter(description = "Type of analysis") @RequestParam(value = "type") ShortCircuitAnalysisType type,
+                                                                            Pageable pageable) {
+        ShortCircuitAnalysisPagedResults pagedResults = shortCircuitService.getPagedResults(resultUuid, mode, type, pageable);
```

Comment:
why this change ?

Reply:
I changed it for 3 reasons : 
- less code : only one endpoint, no duplicates, easier to maintain and understand (for me at least)
- clarity : we know exactly what is sent back for all buses and one bus analysis, without having to check in the front end what we call, and we don't have to call several endpoints for the same result spreadsheet (in one bus, if I followed the current schema, I would have to add one endpoint to get just the associated fault)
- coherence : in all the other MS, we have one endpoint "/result" that send back a DTO, included for pagination (I planned to remove the /paged and the endpoint above not used anymore)

But I understand it's mostly subjective and proabably don't follow the REST best practice.
So I should change it back and use 3 endpoints, including one just to get the associated fault in one bus result ? If so, as discussed yesterday with @SlimaneAmar , we should be coherent in all the Java code and if the rule change then change it in other MS no ?